packages/lesswrong/server/migrations/20231107T124334.ElectionCandidates_tagId_not_null.ts

```diff
@@ -0,0 +1,21 @@
+import ElectionCandidates from "../../lib/collections/electionCandidates/collection";
+
+export const acceptsSchemaHash = "e1db749fcd83e8c45b1b0516537d982b";
+
+export const up = async ({db}: MigrationContext) => {
+  if (ElectionCandidates.isPostgres()) {
+    await db.none(`
+      ALTER TABLE "ElectionCandidates"
+      ALTER COLUMN "tagId" SET NOT NULL
```

Comment:
Changing this wasn't strictly necessary, but I think it's sensible anyway.

We don't currently have utility functions for setting constraints so this is raw SQL - maybe I'll add this in the future but it doesn't seem worth the time now.